fh-mbaas-template-3node.json

```diff
@@ -1677,6 +1677,12 @@
       "value": "info"
     },
     {
+      "name": "MONGODB_PVC_SIZE",
+      "description": "The size of the volume for MongoDB Data",
+      "value": "50Gi",
+      "required": true
+    },
+    {
       "name": "SMTP_SERVER",
```

Comment:
@grdryn I think it is fine to have this parameterized. Our recommendation is 50G but that is very general allowing it to be overridden allows for more flexibility